### PR TITLE
Add: Grid

### PR DESCRIPTION
**Pull request by SJTGSHIVAM**
Added grid component

### CODE REVIEW

File: atoms/grid/index.html [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+          <p class="tui__text-sm tui__m-b-sm">
+            It will represent data in two column format.
+          </p>
+          <ul class="tui__bw-sm tui__pd-md tuid__li--component">
+            <li>
+              <div class="tui__grid-2">
+                <div class="tuid__gird--dispaly"></div>
+                <div class="tuid__gird--dispaly"></div>
+                <div class="tuid__gird--dispaly"></div>
+                <div class="tuid__gird--dispaly"></div>
```
Comment (SJTGSHIVAM) [inline atoms/grid/index.html]: Here is a typo error that we can fix

**Comment by SJTGSHIVAM** [comment]
fixed the mentioned error as well

**Comment by mdarif6** [comment]
Went through the code, and view looking good

File: atoms/grid/index.html [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+      </ul>
+    </main>
+
+    <footer class="tuid__foot"></footer>
+
+    <script src="/utils/preFormatter.js" type="module"></script>
+    <script src="/index.js" type="module"></script>
+    <script src="/utils/prism.js" type="module"></script>
+  </body>
+</html>
```
Comment (mdarif6) [inline atoms/grid/index.html:109]: looking fine

**Review by mdarif6 — APPROVED** [review]

**Review by samyakshah3008 — APPROVED** [review]
Good job, Shivam 
Looks good to go!

**Pull request merged**